charts/topolvm/templates/lvmd/daemonset.yaml

```
@@ -63,8 +63,10 @@ spec:
               command:
                 - /lvmd
                 - health
-            initialDelaySeconds: 10
-            timeoutSeconds: 3
+            failureThreshold: {{ .Values.livenessProbe.csi_registrar.failureThreshold }}
```

This container is lvmd. 
```suggestion
            failureThreshold: {{ .Values.livenessProbe.lvmd.failureThreshold }}
```

---

my bad